testpilot/frontend/static-src/scripts/legal.js

```diff
@@ -0,0 +1,18 @@
+/* global ga*/
+/*eslint-disable*/
+// HACK: Google Analytics lives here, because CSP won't let it live inline
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)
+},i[r].l=1*new Date();a=s.createElement(o),
+m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+/*eslint-enable*/
+
+if (typeof(ga) !== 'undefined') {
+  ga('create', 'UA-49796218-34', 'auto');
+} else {
+  console.warn(
+    'You have google analytics blocked. We understand. Take a ' +
+    'look at our privacy policy to see how we handle your data.'
+  );
+}
```

Comment:
Yeah, it has to be duplicated, because the legal pages don't use the same frontend template as the rest of the site and so don't use the app/main.js code